test/components/kubernetes/kubelet_disruptive_test.go

```diff
@@ -49,6 +49,32 @@ func TestSelfHostedKubeletLabels(t *testing.T) {
 		t.Errorf("could not delete the node %s: %v", chosenNode, err)
 	}
 
+	pc := client.CoreV1().Pods("kube-system")
+	pods, err := pc.List(context.TODO(), metav1.ListOptions{
+		LabelSelector: "k8s-app=kubelet",
```

Comment:
We can use field selector to remove the need to do filtering later: https://github.com/kinvolk/lokomotive/pull/1053/files#diff-26209db90b05e40407e6ff88464ffd88R57-R60

Reply:
Fixed. PTAL.